pkgs/desktops/plasma-5/3rdparty/kwin/scripts/krohnkite.nix

```diff
@@ -22,10 +22,8 @@ mkDerivation rec {
   # 2. plasmapkg2 doesn't copy metadata.desktop into place, so we do that manually
   installPhase = ''
     runHook preInstall
-
-    plasmapkg2 --type kwinscript --install ${src}/res/ --packageroot $out/share/kwin/scripts
-    install -Dm644 ${src}/res/metadata.desktop $out/share/kservices5/krohnkite.desktop
-
+    plasmapkg2 --type kwinscript --install ${src} --packageroot $out/share/kwin/scripts
```

Comment:
Can you move this into a separate commit?

Reply:
That... should not have happened.